tests/unit/schema/test.formats.js

```diff
@@ -123,18 +124,68 @@ describe('formats', () => {
       expect(isAnyUrl(value)).toEqual(true);
       expect(isAbsoluteUrl(value)).toEqual(true);
       expect(isSecureUrl(value)).toEqual(true);
+      expect(isOrigin(value)).toEqual(true);
 
       expect(isStrictRelativeUrl(value)).toEqual(false);
       expect(imageDataOrStrictRelativeUrl(value)).toEqual(false);
     });
 
+    it('trailing slash', () => {
```

Comment:
would be good to put actual descriptions/sentences here and in the other _added_ test cases, otherwise the output will look like:

```
it trailing slash
it domain with no path but with query string
```

Reply:
I did this in the other file, but here I was trying to stay consistent on the other URL formats tests, plus those tests make several assertions (some true, some false) so I wasn't sure what to do...

Reply:
Yeah, it is quite involved actually so let's forget about this suggestion.